web/themes/custom/vote_gov/php-includes/form.inc

```diff
@@ -36,7 +36,7 @@ function vote_gov_form_alter(&$form, FormStateInterface $form_state, $form_id):
   }
 
   // Add the USWDS form class.
-  $form['#attributes']['class'][] = 'usa-form';
+  $form['#attributes']['class'] = 'usa-form';
```

Comment:
It appears like we are replacing a class array with a string. What is the expected result?

Reply:
This resolved the following error when loading the login form at /user/login
Error: [] operator not supported for strings in vote_gov_form_alter() (line 39 of themes/custom/vote_gov/php-includes/form.inc).

The class should be set on the login form itself.

Reply:
I found the issue it was on line 62 of the usagov_login.module file. I was able to update and revert the theme file change.